Gopkg.toml

```diff
@@ -72,3 +72,7 @@ required = [
 [[constraint]]
   name = "github.com/onsi/ginkgo"
   version = "1.5.0"
+
+[[constraint]]
+  name = "gopkg.in/yaml.v2"
```

Comment:
This one was unfavoured for a long time, but perhaps v2 has no major issue... however, I think it's better to stick to `"github.com/ghodss/yaml"`, which is what Kubernetes uses.

Reply:
I'll change this.